components/cell/jwks-server/crypto/resolver/key_resolver_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package resolver
+
+import (
+	"crypto/tls"
+	//"github.com/cellery-io/mesh-security/components/cell/jwks-server/crypto/service"
+	"github.com/cellery-io/mesh-security/components/cell/jwks-server/resources"
+	"io/ioutil"
+	"log"
+	"testing"
+)
+
+const keyFilePath string = "../../resources/key.pem"
+const certFilePath string = "../../resources/cert.pem"
+
+func TestFileBasedKeyPair(t *testing.T) {
+	keyData, err := ioutil.ReadFile(keyFilePath)
+	certData, err := ioutil.ReadFile(certFilePath)
+
+	if err != nil {
+		log.Printf("Error reading the files. %s", err)
```

Comment:
Overall we can improve logging messages. Can we do it ?. We need to have a better context to identify. Usually it's not only the developer who will read logs and try to diagnose.